rust/cardano-chain-follower/src/mithril_turbo_downloader.rs

```diff
@@ -366,28 +358,81 @@ impl SnapshotDownloader for MithrilTurboDownloader {
             new_files,
         );
 
-        debug!("Download and Unpack finished='{location}' to '{target_dir:?}'.");
```

Comment:
dont remove debug statements.

Reply:
This debug! statement I have not removed, its just nor correctly showed in the diff.

 look at this line
https://github.com/input-output-hk/catalyst-libs/pull/309/files#diff-c00feecb5182da50ec77b745d2b9950c1925221b805111327072a317f24c0c7eR384